src/shader.cpp

```
@@ -119,62 +121,126 @@ Result ShaderProgram::compileShaders(Device* device)
+            entryPoint.stats.endTime,
+            entryPoint.stats.totalTime,
+            entryPoint.stats.downstreamTime,
+            entryPoint.stats.isCached,
+            entryPoint.stats.cacheSize
+        );
+    }
+}
+
+Result ShaderProgram::installCompiledEntryPoints(std::span<const CompiledEntryPoint> entryPoints)
+{
+    for (const auto& entryPoint : entryPoints)
```

Cppcheck (2.21.0)

[style] The function 'rbegin' is never used.

(unusedFunction)